build.gradle

```diff
@@ -7,22 +11,18 @@ group = 'ARIN'
 version = '1000.0-SNAPSHOT'
 description = """RDAP Bootstrap Server"""
 
-sourceCompatibility = 1.6
-targetCompatibility = 1.6
```

Comment:
Is it worth being explicit about a sourceCompatibility = '11'?

Reply:
This is a good idea.

Reply:
Done.